src/control/__tests__/configureIndex.test.ts

```diff
@@ -25,21 +25,35 @@ describe('configureIndex', () => {
         ready: true,
         state: 'Ready',
       },
+      tags: {
+        example: 'tag',
+      },
+      deletionProtection: 'disabled', // Redundant, but for example purposes
```

Comment:
Added `deletionProtection` to this test just b/c I noticed it was missing while doing index tag stuffs